modules/hostap/src/supp_events.h

```diff
@@ -23,7 +23,9 @@ enum net_event_wpa_supp_cmd {
 	NET_EVENT_WPA_SUPP_CMD_IFACE_ADDED,
 	NET_EVENT_WPA_SUPP_CMD_IFACE_REMOVING,
 	NET_EVENT_WPA_SUPP_CMD_IFACE_REMOVED,
-	NET_EVENT_WIFI_CMD_MAX
+	NET_EVENT_WPA_SUPP_CMD_COUNT,
+	NET_EVENT_WPA_SUPP_PRIV_START = NET_EVENT_WPA_SUPP_CMD_COUNT,
```

Comment:
We don't need `priv` concept here, its only needed to abstract NM in upstream.

Reply:
Removed priv.